persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_16/main.bal

```diff
@@ -0,0 +1,15 @@
+import ballerina/io;
```

Comment:
Add license headers

Reply:
Updated in [704a43f](https://github.com/ballerina-platform/persist-tools/pull/46/commits/704a43f3344de31ca790267f6ba9cc31ad09d26f)